packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -431,17 +448,39 @@ export const transformPlotlyJsonToVBCProps = (
       return;
     }
 
-    const isXString = isStringArray(series.x);
+    const xValues: (string | number)[] = [];
+    const yValues: number[] = [];
+    series.x.forEach((xVal, index) => {
+      if (isInvalid(xVal)) {
+        return;
+      }
+
+      let yVal: number;
+      if (Array.isArray(series.y)) {
+        if (isInvalid(series.y[index]) || typeof series.y[index] !== 'number') {
+          return;
+        }
+
+        yVal = series.y[index] as number;
+      } else {
+        yVal = 1;
```

Comment:
convert this into a const giving it appropriate name

Reply:
done